util/metrics/src/service_metrics.rs

```diff
@@ -195,6 +203,7 @@ fn path_from_ctx(ctx: &RpcContext) -> Option<String> {
 
 /// This method reads the full URI from gRpcContext (looks like:
 /// `/{package}.{service_name}/{method}`
+/// ('/' equates to ascii code 47)
 /// and converts it into a dot-delimited string, dropping the 1st `/`
```

Comment:
nit: it feels like the notation about the ascii code interrupts the thought.
```suggestion
/// and converts it into a dot-delimited string, dropping the 1st `/`
/// ('/' equates to ascii code 47)
```

Reply:
Same ?